.python-version

```diff
@@ -0,0 +1 @@
+3.5.2
```

Comment:
Just confirmed that cloud.gov is using the latest Python buildpack (via the `cf buildpacks` command, which returns [1.5.14](https://github.com/cloudfoundry/python-buildpack/releases/tag/v1.5.14) and is the current latest release).

Like I said, I think it makes more sense to move this work out into its own PR.  How do others feel about that?